back-nest/package.json

```diff
@@ -17,15 +17,15 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "migrate": "npx prisma migrate dev"
```

Comment:
I think you forgot to include `prisma generate` command. It [reads your schema](https://docs.nestjs.com/recipes/prisma#install-and-generate-prisma-client) and updates the typings for typescript in node_modules folder.

Reply:
Also, don't forget to update Readme. Include all commands step by step there. Dont forget to update Docker commands to start up the application as well (if needed)